js/src/index.ts

```diff
@@ -50,6 +54,46 @@ export class Schematic {
     this.apiKey = apiKey;
     this.eventQueue = [];
     this.useWebSocket = options?.useWebSocket ?? false;
+    this.debugEnabled = options?.debug ?? false;
+    this.offlineEnabled = options?.offline ?? false;
+
+    // Check for debug mode in URL query parameter
+    if (
+      typeof window !== "undefined" &&
+      typeof window.location !== "undefined"
+    ) {
+      const params = new URLSearchParams(window.location.search);
```

Comment:
I'm curious if this will actually work in all scenarios -- i wouldn't be surprised if ppl have url rewriting stuff that can remove these before we load. but also, this is one way to trigger it so it's not that big of a deal.

Reply:
I agree this won't always work